### PR TITLE
Upgraded Formik to 2.0.6

### PR DESCRIPTION
**Pull request by bhardwajamit0890**

### CODE REVIEW

File: lib/components/molecules/Form/Form.story.js [hunk 1]
```diff
@@ -27,7 +27,7 @@ storiesOf('Molecules', module)
             sectionFn: () => (
               <Form {...userFormProps}>
                 <div>
-                  <Form.Select {...userFormSelectProps} />
+                  <Form.Select {...userFormSelectProps}></Form.Select>
```
Comment (vinodloha) [inline lib/components/molecules/Form/Form.story.js:30]: Doesn't need a closing tag

File: lib/components/molecules/Form/FieldTextarea/FieldTextarea.js [hunk 2]
```diff
@@ -49,7 +47,7 @@ const FieldTextarea = ({
           />
         );
       }}
-    />
+    </Field>
```
Comment (vinodloha) [inline lib/components/molecules/Form/FieldTextarea/FieldTextarea.js:50]: Doesn't need a closing tag

**Review by vinodloha — APPROVED** [review]

**Review by vinodloha — COMMENTED** [review]
@bhardwajamit0890 can you fix the breaking build?

**Comment by bhardwajamit0890** [comment]
NEw PR raised closing this. https://github.com/pagesource/atomic-react-components/pull/148

**Pull request closed without merging**